SPQR.py

```diff
@@ -0,0 +1,44 @@
+# # Generates sighash values from available information on a raw transaction,
```

Comment:
Can we `git mv` this into the tests directory? Maybe make `riemann/tests/scripts`